kanzucode/angie-server
# Ref-014, As MC admin, I can see which assigned members have attended their first MC meeting

**What does this PR do?**

- This PR adds a feature that allows MC admins to view first time guests. These are members who have attended an MC event for the first time.

**Description of tasks?**

- MC admins are now able to see new members who have attended an MC event.

**How to manually test this?**

- Under Reports, select a particular report to view then select members who have attended the MC. There should be a “First Time Guests” tab next to it to show members who have attended for the first time.

[Link to Client PR](https://github.com/kanzucode/angie-client/pull/50#issue-624690985)
